src/app/crisis-helpline/crisis-helpline.component.html

```diff
@@ -36,7 +36,7 @@ <h1 class="title">Crisis Helpline</h1>
     </div>
 
     <div class="item-1">
-      <h2>Lorem Ipsum Lorem Ipsum</h2>
+      <!-- <h2>Lorem Ipsum Lorem Ipsum</h2> -->
```

Comment:
Should we remove this or what?

Reply:
I think we should remove it.

Reply:
```suggestion
      <h2>Virtual Emergency Support</h2>
```

Nah, I think the heading here is supposed to match the side-menu.